.travis.yml

```diff
@@ -11,7 +11,13 @@ install:
 
 services:
   - mongodb
-# make mongodb a replica set
+ddons:
```

Comment:
You're missing a letter here. Also, if you're installing mongodb-org-server via addons you don't need to enable the mongodb service as those will conflict.